vue/src/views/Dashboard.vue

```diff
@@ -18,40 +18,60 @@ export default {
   components: {
     Standard,
   },
-  data() {
-    return {
-      standards: _times(16, key => ({
-        key,
-        organization: {
-          logo: LOGO_WIN,
-          name: 'WIN',
-        },
-        occupation: {
-          name: 'Mechatronics Technician',
-          type: 'Hybrid',
-          onet: '51-4012.00',
-          cb: '1100CB',
-        },
-        workProcesses: _times(18, () => ({
-          skills: _times(8, () => ({})),
-          hoursTotal: 334,
-        })),
-      })),
-    };
+  computed: {
+    standards() {
+      // TODO: remove fake data
```

Comment:
@jhash From the title of this PR, I thought this would remove the fake data. Either way, make sure you highlight gaps in what we're returning? I don't know if we've covered everything in the designs (organization logo, for example).

Reply:
@Pcushing I'm working with @littleforest now to find the missing data